freqtrade/exchange/common.py

```diff
@@ -121,19 +121,30 @@ def decorator(f):
         @wraps(f)
         def wrapper(*args, **kwargs):
             count = kwargs.pop('count', retries)
+            kucoin = args[0].name == "Kucoin"  # Check if the exchange is KuCoin.
             try:
                 return f(*args, **kwargs)
             except (TemporaryError, RetryableOrderError) as ex:
                 msg = f'{f.__name__}() returned exception: "{ex}". '
                 if count > 0:
-                    logger.warning(msg + f'Retrying still for {count} times.')
+                    msg += f'Retrying still for {count} times.'
                     count -= 1
                     kwargs.update({'count': count})
                     if isinstance(ex, (DDosProtection, RetryableOrderError)):
-                        # increasing backoff
-                        backoff_delay = calculate_backoff(count + 1, retries)
-                        logger.info(f"Applying DDosProtection backoff delay: {backoff_delay}")
-                        time.sleep(backoff_delay)
+                        if kucoin and "429000" in str(ex):
```

Comment:
You may want to add a test for this (similar to `test__async_kucoin_get_candle_history` - but for a sync endpoint).

While the code does work now - that can't be guaranteed for the future without test - and especially in this area of the bot, it's critical to be able to rely on the code to work correctly.